src/LCM2ROSControl.hpp

```diff
@@ -40,6 +44,14 @@ namespace valkyrie_translator
     double ff_qd_d;
     double ff_f_d;
     double ff_const;
+  } joint_gains;
```

Comment:
`typedef struct _bla { } bla;` is the C way of declaring structs. in C++ you can just use `struct bla { };`